kurtosis-module/Dockerfile

```diff
@@ -27,9 +27,6 @@ WORKDIR /run
 # Copy the code into the container
 COPY --from=builder /build/kurtosis-module.bin .
 
-# Copy local static files inside the Kurtosis module image
-COPY kurtosis-module/static_files /static-files
```

Comment:
Why remove this?

Reply:
ups! it seems that I was very motivated removing things that I forgot that this directory is a default kurt module directory. I'll fix this.